internal/services/eventhub/eventhub_namespace_resource_test.go

```diff
@@ -398,14 +400,7 @@ func TestAccEventHubNamespace_BasicWithSkuUpdate(t *testing.T) {
 				check.That(data.ResourceName).Key("capacity").HasValue("2"),
 			),
 		},
-		{
-			Config: r.premium(data),
-			Check: acceptance.ComposeTestCheckFunc(
-				check.That(data.ResourceName).ExistsInAzure(r),
-				check.That(data.ResourceName).Key("sku").HasValue("Premium"),
-				check.That(data.ResourceName).Key("capacity").HasValue("1"),
-			),
-		},
```

Comment:
I would like to understand what the failure is here, is this triggering a ForceNew or does the API forbid this?

Reply:
It's triggering ForceNew
```
=== RUN   TestAccEventHubNamespace_BasicWithSkuUpdate
=== PAUSE TestAccEventHubNamespace_BasicWithSkuUpdate
=== CONT  TestAccEventHubNamespace_BasicWithSkuUpdate
    testcase.go:173: Step 3/3 error: Pre-apply plan check(s) failed:
        'azurerm_eventhub_namespace.test' - expected action to not be Replace, path: [[sku]]
```